pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
```

Comment:
Also using [pnpm-workspace](https://pnpm.io/workspaces) since its probably going to be easier and faster to install now